index.js

```diff
@@ -74,6 +74,10 @@ module.exports = function(options) {
     }
 
     this.body = res.body;
+
+    if (options.yieldNext) {
+      yield next;
```

Comment:
By the way, I'm not sure if this should be `yield` or a `yield*`. Both seem to work. I kept `yield` as that's what was in https://github.com/popomore/koa-proxy/pull/16 but I can change it if it should be `yield*`